precompile/contract_native_minter.go

```diff
@@ -64,6 +70,13 @@ func (c *ContractNativeMinterConfig) Address() common.Address {
 
 // Configure configures [state] with the desired admins based on [c].
 func (c *ContractNativeMinterConfig) Configure(_ ChainConfig, state StateDB, _ BlockContext) {
+	for to, amount := range c.InitialMint {
+		if amount != nil {
+			bigIntAmount := (*big.Int)(amount)
+			state.AddBalance(to, bigIntAmount)
+		}
+	}
+
```

Comment:
should check `InitialMint` in Equals as well

Reply:
yeah nice catch 👍